docs/getting-started.md

```diff
@@ -172,3 +173,215 @@ $ curl -i localhost:8080/httpbin/get
+  active: true
+  proxy:
+    target_url: http://httpbin.org
+    listen_path: /httpbin
+    strip_listen_path: true
+```
+
+and apply changes :
+```bash
+kubectl apply -f config/samples/httpbin.yaml
+
+apidefinition.tyk.tyk.io/httpbin configured
+```
+
+Or,
+
+```bash
+cat <<EOF | kubectl apply -f -
+apiVersion: tyk.tyk.io/v1alpha1
+kind: ApiDefinition
+metadata:
+    name: httpbin
+spec:
+    name: httpbin
+    use_keyless: false
+    protocol: http
+    active: true
+    proxy:
+        target_url: http://httpbin.org
+        listen_path: /httpbin
+        strip_listen_path: true
+EOF
+
+apidefinition.tyk.tyk.io/httpbin configured
+```
+
+In order to access `httpbin` API, you need to include a key to the header.
+
+> **Note**: All supported authentication types by Tyk Operator are listed [here](https://github.com/TykTechnologies/tyk-operator/blob/master/docs/api_definitions.md#client-to-gateway-authentication).
+Authentication token is the default one if you are using keyless API.
+
+```bash
+curl -i localhost:8080/httpbin/get
+HTTP/1.1 401 Unauthorized
+Content-Type: application/json
+X-Generator: tyk.io
+Date: Thu, 03 Mar 2022 15:47:30 GMT
+Content-Length: 46
+
+{
+    "error": "Authorization field missing"
+}%
+```
+
+We need to generate a key to access the `httpbin` API. Generating a key depends on which type of Tyk installation you have.
+
+### Tyk CE
+
+To create an API Key, we will need the API ID that we wish to grant the key access to, then creating the key is a very simple API call to the endpoint.
+
+> **Prerequisite**: You will need your API secret, this is the `secret` property of the tyk.conf file.
+Once you have this value, you can use them to access the Gateway API
+
+In order to obtain API ID, we can describe ApiDefinition resource that represents our `httpbin` API.
+
+```bash
+kubectl describe tykapis httpbin
+Name:         httpbin
+Namespace:    default
+Labels:       <none>
+Annotations:  <none>
+API Version:  tyk.tyk.io/v1alpha1
+Kind:         ApiDefinition
+Metadata:
+  ...
+Spec:
+  ...
+  api_id:  ZGVmYXVsdC9odHRwYmlu
+  ...
+Status:
+  api_id:  ZGVmYXVsdC9odHRwYmlu
+Events:    <none>
+```
+
+Once you describe `httpbin` ApiDefinition, the API ID is represented in the `status.api_id` field.
+
+In our example,
+- `{API-NAME}`: httpbin
+- `{API-ID}`: ZGVmYXVsdC9odHRwYmlu
+
+The below curl command will generate a key for our API:
+
+```bash
+curl -X POST -H "x-tyk-authorization: {API-SECRET}" \
+  -s \
+  -H "Content-Type: application/json" \
+  -X POST \
+  -d '{
+    "allowance": 1000,
+    "rate": 1000,
+    "per": 1,
+    "expires": -1,
+    "quota_max": -1,
+    "org_id": "1",
+    "quota_renews": 1449051461,
+    "quota_remaining": -1,
+    "quota_renewal_rate": 60,
+    "access_rights": {
+      "{API-ID}": {
+        "api_id": "{API-ID}",
+        "api_name": "{API-NAME}",
+        "versions": ["Default"]
+      }
+    },
+    "meta_data": {}
+  }' http://localhost:8080/tyk/keys/create | python -mjson.tool
+{
```

Comment:
Move response in separate section so user is able to copy just the curl request.